tests/test_key_revocation.py

```diff
@@ -385,9 +388,6 @@ def test_root_key_revocation(self):
       for mirror_exception in exception.mirror_errors.values():
         self.assertTrue(isinstance(mirror_exception, tuf.BadSignatureError))
 
-    # Load the previous Root signing key so that the the client can update
-    # successfully.
-    repository.root.load_signing_key(self.role_keys['root']['private'])
```

Comment:
FYI
To prevent breaking the chain of subsequent versions of root, here is the solution that I implemented...

(1) Add a method to allow individual roles to be written and (2) add a flag to this method to control whether version numbers are automatically incremented.  In this way, we can write the temporary root file (with the missing, required root key) for testing purposes and then regenerate the valid root file with the expected version number and signatures.

For example (Note, some steps removed to keep the example small):
```Python
repository.root.remove_verification_key(public_root_key)
repository.writeall()
# The writeall() above automatically increments root to version 2 (supposing it's
# currently at version 1.

# Verify that the client rejects the invalid root file....
try:
  updater.refresh()
except tuf.BadSignatureError:
  ...

# Now regenerate a valid root file, where its version number = 2
repository.root.add_verification_key(public_root_key)
repository.write('root', increment_version_number=False)                    
repository.write('snapshot')                                                
repository.write('timestamp')

# Successfully refresh root...
updater.refresh()
self.assertEqual(updater.metadata['current']['root']['version'], 2)
```